build.gradle

```diff
@@ -210,6 +210,27 @@ test {
     systemProperties('spock.skip.sandbox': System.getProperty('spock.skip.sandbox', 'true'))
 }
```

Comment:
just let test depend the sub tasks e.g. 
```
test {
    dependsOn(unitTest, integrationTest)
}
```

so there is no duplication of definitions

Reply:
resolved in [d8e31fb](https://github.com/jenkinsci/ecu-test-execution-plugin/pull/118/commits/d8e31fbccc38836a7f79bd29155a290e66e8f93d)